lib/pc_api_internal.h

```diff
@@ -299,7 +301,9 @@ void pc_bitmap_filter(PCBITMAP *map, PC_FILTERTYPE filter, int i, double d, doub
 */
 
 void pc_matrix_set_from_quaternion(PCMAT33 mat, double qw, double qx, double qy, double qz);
+void pc_matrix_set_affine(PCMAT43 mat, double a, double b, double c, double d, double e, double f, double g, double h, double i, double xoff, double yoff, double zoff);
 void pc_matrix_multiply_vector(PCVEC3 rotatedvec, const PCMAT33 mat, const PCVEC3 vec);
+void pc_matrix_multiply_vector_43(PCVEC3 res, const PCMAT43 mat, const PCVEC4 vec);
```

Comment:
No, I don't think so. It needs to be a 4-dimension vector with the last coordinate set to 1.